src/ResourceManager/Websites/Commands.Websites.Test/Commands.Websites.Test.csproj

```diff
@@ -325,6 +319,10 @@
       <Project>{80a92297-7c92-456b-8ee7-9fb6ce30149d}</Project>
       <Name>Commands.Websites</Name>
     </ProjectReference>
+    <ProjectReference Include="E:\FileStore\Code\azure-sdk-for-net\src\SDKs\WebSites\Management.Websites\Microsoft.Azure.Management.Websites.csproj">
```

Comment:
@Nking92 I would recommend building an unsigned package using `dotnet pack` from the SDK repo and referencing that `.nupkg` file instead of the project. This will help with build failures.

Reply:
Should I add that .nupkg to the commit as well? I was assuming it wouldn't pick up the reference until we publish our new SDK version.

Reply:
@Nking92 you can place the `.nupkg` file inside of the `tools/LocalFeed` folder - the reference will get picked up there